test/unit/spread.spec.ts

```diff
@@ -87,3 +87,22 @@ describe("in object literal", () => {
         `.expectToMatchJsResult();
     });
 });
+
+const spreadCases = [
+    "1, 2, ...[3, 4, 5]",
+    "...[1, 2], 3, 4, 5",
+    "1, 2, ...'spread', 4, 5",
+    "1, [2], ...[3, 4, 5]",
+    "...[1, 2, 3], 4, ...[5, 6]",
+];
+
+test.each(spreadCases)("spread equivalence for array elements (%p)", expression => {
+    util.testExpression`[${expression}]`.expectToMatchJsResult();
+});
+
+test.each(spreadCases)("spread equivalence for call arguments (%p)", expression => {
```

Comment:
Like with `in object literal/of object literal`, I think that should be tested as a part of `in function|array call/of array literal`